### DISCUSSION TITLE
Sceen shareing

### DISCUSSION THREAD

**Discussion by EgeresiMihaly** [stoatchat/discussions#719] [Feature Suggestions]
it would be nice to have a screen sharing feature and the mobile notifications are pretty much not coming only if you have to enable it in the app settings it is blocked by default so the message notifications come who wrote but when I go to it the app doesn't open but the web page of the app so not the best you can't do an update for that otherwise apart from these pretty good app you are heading in the right direction keep it up

**Comment by xibn** [comment]
To be honest, I'm not sure if streaming is in the budget right now.
